From 7097de23c5d93ef33db583615c0373aebf75333c Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Mon, 10 Aug 2015 20:47:37 +0200
Subject: [PATCH] don't overwrite CXXFLAGS

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)


diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0ebf352..ebbf68f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ find_package(Arcus REQUIRED)
 if(NOT ${CMAKE_VERSION} VERSION_LESS 3.1)
     set(CMAKE_CXX_STANDARD 11)
 else()
-    set(CMAKE_CXX_FLAGS "-std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
 # Add warnings
