From 29d933476e8bff19b52dcdb1e2b958d2346b094d Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Sat, 28 Nov 2015 11:54:29 +0100
Subject: [PATCH] setup: lock netifaces to 0.10.4

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 86b7837..bc9513e 100644
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,7 @@ INSTALL_REQUIRES = [
 	"netaddr==0.7.17",
 	"watchdog==0.8.3",
 	"sarge==0.1.4",
-	"netifaces==0.10",
+	"netifaces==0.10.4",
 	"pylru==1.0.9",
 	"rsa==3.2",
 	"pkginfo==1.2.1",
-- 
2.0.1

